tools/python/CMakeLists.txt

```diff
@@ -27,13 +27,6 @@ if (POLICY CMP0042)
 endif()
 
 
-# To avoid dll hell, always link everything statically when compiling in
-# visual studio.  This way, the resulting library won't depend on a bunch
-# of other dll files and can be safely copied to someone else's computer
-# and expected to run.
-if (MSVC)
-    include(${CMAKE_CURRENT_LIST_DIR}/../../dlib/cmake_utils/tell_visual_studio_to_use_static_runtime.cmake)
-endif()
```

Comment:
This makes building fail in the very latest version of visual studio and the python interpreter in windows now.